src/common/io.ts

```diff
@@ -824,6 +824,39 @@ export class IOWriteable extends IOReadable implements AoIOWrite {
     params: {
       target: string;
       decreaseQty: number | mIOToken;
+      instant: boolean;
```

Comment:
i would make this optional - and default to false
```suggestion
      instant?: boolean;
```

Reply:
Fixed 

https://github.com/ar-io/ar-io-sdk/commit/ae7be5caaa79f470eb9755c05c85467e98643d55